app/assets/javascripts/components/newUser.jsx

```diff
@@ -0,0 +1,53 @@
+class CreateUser extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.clearError = this.clearError.bind(this)
+    this.state = {
+      isChecked: false,
+      showFailed: false,
+      errorMessage: ''
+    }
+  }
+
+  renderSignUp() {
```

Comment:
Where are you calling this? Also, we don't let users sign up now. So it's more of a pure create user.

Reply:
It is for creating users. Will take it off in this PR and include it in the next for modularity.